docs/source/using_pyumbral.rst

```diff
@@ -137,22 +129,21 @@ or re-encrypted for him by Ursula, he will not be able to open it.
 
 .. doctest:: capsule_story
 
-    >>> fail = pre.decrypt(ciphertext=ciphertext,
-    ...                    capsule=capsule,
-    ...                    decrypting_key=bobs_private_key)
+    >>> fail = decrypt_original(sk=bobs_private_key,
+    ...                         capsule=capsule,
+    ...                         ciphertext=ciphertext)
     Traceback (most recent call last):
         ...
-    umbral.pre.UmbralDecryptionError
+    umbral.dem.ErrorInvalidTag
```

Comment:
wdyt about `InvalidTagError` or just `InvalidTag` ? 😎

Reply:
Yes, the what errors to raise is still something I'm thinking about. Should ideally be synchronized with rust-umbral too.

Reply:
Now an `umbral.UmbralError` is raised.